src/main/java/com/github/wovnio/wovnjava/QueryPatternHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.regex.Pattern;
+
+class QueryPatternHandler extends PatternHandler {
+    static final String QueryPatternRegex = "(?:(?:\\?.*&)|\\?)wovn=([^&]+)(?:&|$)";
+
+    QueryPatternHandler() {
+        this.getLangPattern = Pattern.compile(QueryPatternRegex);
+    }
+
+    String getLang(String url) {
+        return this.getLangMatch(url, this.getLangPattern);
+    }
+
+    String removeLang(String url) {
+        return "site.com/path";
```

Comment:
PR is WIP?

Reply:
As the PR description indicates, this is the first of several PRs related to URL management refactoring that I will all merge into `refactor/url_management` before releasing.

In this PR, only the `getLang` operation is migrated to the PatternHandlers. Its behavior from before remains unchanged. The other PatternHandler functions are placeholders for now.

Reply:
I saw that afterwards. Test would be good for implemented stuffs tho.